custom_components/tuya_local/devices/tefal_xplorer_vacuum.yml

```diff
@@ -0,0 +1,295 @@
+name: Tefal vacuum
+products:
+  - id: gahugwuufkhzie7v
+    name: Tefal X-plorer Serie 75 Animal
+primary_entity:
+  entity: vacuum
+  dps:
+    - id: 101
+      name: activate
+      type: boolean
+    - id: 102
+      name: pause
+      type: boolean
+      hidden: true
+    - id: 104
+      name: command
+      type: string
+      mapping:
+        - constraint: pause
+          conditions:
+            - dps_val: true
+              value: pause
+            - dps_val: false
+              mapping:
+                - dps_val: smart
+                  value: smart
+                - dps_val: pose
+                  value: clean_spot
+                - dps_val: zone
+                  value: zone
+                - dps_val: backcharge
+                  value: return_to_base
+                - dps_val: curpointing
+                  value: curpointing
+                - dps_val: selectroom
+                  value: selectroom
+    - id: 105
+      name: status
+      type: string
+      mapping:
+        - dps_val: idle
+          value: standby
+        - dps_val: pointing\areaing
+          value: pointing_areaing
+        - dps_val: totaling
+          value: cleaning
+        - dps_val: sweep
+          value: sweep
+        - dps_val: mop
+          value: mop
+        - dps_val: fault
+          value: error
+        - dps_val: pause
+          value: paused
+        - dps_val: chargring
+          value: charging
+        - dps_val: tocharge
+          value: returning
+        - dps_val: fullcharge
+          value: charged
+        - dps_val: remotectl
+          value: remotectl
+        - dps_val: dormant
+          value: dormant
+        - dps_val: curpointing
+          value: curpointing
+        - dps_val: selectroom
+          value: selectroom
+    - id: 106
+      name: battery_level
```

Comment:
battery_level is deprecated for the vacuum platform. A separate sensor should be used instead.

Reply:
but the vacuum cleaner itself has this information, and thuja returns it (viewing all DPs via ioBroker)

![image](https://github.com/make-all/tuya-local/assets/8526410/83577965-37d6-470a-ac24-657de3ae48b8)

Reply:
You can, of course, remove it, but then it will not be displayed in the vacuum_card

![image](https://github.com/make-all/tuya-local/assets/8526410/41595834-323c-432f-88fd-2c7cc3e9b56a)

Reply:
I am not talking about removing it, but moving it to a secondary entity of type sensor, with class battery.  The standard HA vacuum more-info dialog supports that, though some custom cards have not yet been updated for this change from last July. https://developers.home-assistant.io/blog/2023/07/10/vacuum-updates/

Reply:
I got it, thanks, I corrected it